1001-2000/LIT1803Lifeof.xml

```diff
@@ -31,7 +31,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </sourceDesc>
          <sourceDesc>
             <listWit>
-               <witness corresp="BNFabb60"/>
+               <witness corresp="BNFabb60">BnF d'Abb. 60, f. 1r-16r</witness>
```

Comment:
Thank you Marcin. This is very good in principle, just I do not know if this way of encoding is used elsewhere. Perhaps @eu-genia can tell something more.
In any case, what about `<witness corresp="BNFabb60#i1"/>`? Does this point to the exact textual unit?

Reply:
`<witness corresp="BNFabb60#i1"/>` would not print anything ) first, `i1 `does not exist, second, the app can only print the exact shelfmark if only the exact shelfmark is in `corresp`, if you want to link to an item you must provide the text 

so it is OK to list
`<witness corresp="ID"/> `
or 
`<witness corresp="ID#ms_i1">ms Full shelfmark, ff. xx-yy</witness>`

In this case the manuscript does not appear only because there is no item BNFabb60 in the application. the ID must be corrected to point to an existing record

Reply:
(or a record for BNFabb60 must be created, then no correction in this LIT record would be necessary at all)